File: desktop.py
#!/usr/bin/env python3

from applescript import AppleScript as applescript
from types import SimpleNamespace

def position():
	script = 'tell application "Finder" to get bounds of window of desktop'
	dimensions = applescript(script).run()

	return SimpleNamespace(**{
		"left": dimensions[0],
		"top": dimensions[1],
		"width": dimensions[2],
		"height": dimensions[3]
	})

print(position())

File: desktop.swift
#!/usr/bin/env xcrun swift

import Cocoa
import Foundation

func printInfo(_ value: Any) {
    let t = type(of: value)
    print("'\(value)' of type '\(t)'")
}

var frame = NSScreen.screens[0].frame

var screen = [
	"width": String(format: "%.0f", frame.width),
	"height": String(format: "%.0f", frame.height)
]

print(screen)
print(screen["width"]!, screen["height"]!)
